### PR TITLE
fix content guideline copy

### LINKED ISSUE #343: Change "MIT Content Policy" to "MicroMasters Discussions Community Guidelines"
Changes at https://github.com/mitodl/open-discussions/blob/b5e4c60677024eb6bc5d04cb86f1f7ee7ef3c41a/static/js/containers/ContentPolicyPage.js#L14 and https://github.com/mitodl/open-discussions/blob/b5e4c60677024eb6bc5d04cb86f1f7ee7ef3c41a/static/js/components/CreatePostForm.js#L139

### PR DESCRIPTION
**Pull request by aliceriot**
#### What are the relevant tickets?

closes #343

#### What's this PR do?

this just makes a quick change to the page title and the text of one link.

#### How should this be manually tested?

Just make sure the changes show up.

### CODE REVIEW

File: static/js/containers/ContentPolicyPage.js [hunk 1]
```diff
@@ -11,7 +11,9 @@ export default class ContentPolicyPage extends React.Component<*, void> {
   render() {
     return (
       <div className="content">
-        <DocumentTitle title={formatTitle("Content Policy")} />
+        <DocumentTitle
+          title={formatTitle("MicroMasters Discussions Community Guidelines")}
+        />
         <div className="main-content">
           <Card className="content-policy">
             <h1>MicroMasters Discussion Guidelines</h1>
```
Comment (noisecapella) [inline static/js/containers/ContentPolicyPage.js]: Should we update this also to `MicroMasters Discussion Community Guidelines`?

File: static/js/containers/CreatePostPage_test.js [hunk 2]
```diff
@@ -60,7 +60,8 @@ describe("CreatePostPage", () => {
           actions.forms.FORM_BEGIN_EDIT,
           actions.channels.get.requestType,
           actions.channels.get.successType,
-          actions.subscribedChannels.get.requestType
+          actions.subscribedChannels.get.requestType,
+          actions.subscribedChannels.get.successType
```
Comment (noisecapella) [inline static/js/containers/CreatePostPage_test.js]: :+1:

File: static/js/containers/ContentPolicyPage.js (hunk 1 above)
Comment (aliceriot) [reply static/js/containers/ContentPolicyPage.js]: @pdpinch

Comment (pdpinch) [reply static/js/containers/ContentPolicyPage.js]: thanks for catching. Yes please.

**Pull request merged**